### DISCUSSION TITLE
On Windows, how to send Windows key for SUPER?

### DISCUSSION THREAD

**Discussion by valankar** [wezterm/wezterm#3245] [Q&A] [answered]
Probably dumb question, but the Windows key sends to the OS, and I'm not sure how to use the SUPER modifier. Is there something I need to do to get that working? Thanks.

**Comment by wez** [comment]
wezterm can only act on key events that are delivered to it, so if your OS/windowing environment doesn't pass on SUPER then wezterm cannot do anything about it.

You may be able to use some other Windows software to influence the system, but I don't have any specific recommendations.

**Comment by valankar** [comment]
It's strange, some of the combinations do work. SUPER-n does create a new window.

However, SUPER-1 launches task manager. I figure there is some Windows configuration setting for this.

**Comment by valankar** [comment, marked as answer]
Ah, the issue is if there are apps pinned to the taskbar, WINDOWS-# keys will start those apps. There seems to be some registry changes to turn that off:

https://answers.microsoft.com/en-us/windows/forum/all/how-to-turn-off-windows-shortcut-keys-windows-11/9963a05a-3518-404b-a0cb-3c9ab887e681

I'll stick with maybe remapping these keys.

**Comment by bzm3r** [comment]
Yeah, disabling it via GPMSC doesn't really work. It seems there are additional settings (don't know which ones) which also need to be changed. For example, on Windows 10: Win + W always triggers a stupid whiteboard popup. 

I am going to try playing around with [`kanata`](https://github.com/jtroo/kanata) to help me out here.
